server/etcdserver/api/etcdhttp/health.go

```diff
@@ -21,13 +21,14 @@ import (
 	"net/http"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/zap"
+
 	"go.etcd.io/etcd/api/v3/etcdserverpb"
 	pb "go.etcd.io/etcd/api/v3/etcdserverpb"
 	"go.etcd.io/etcd/client/pkg/v3/types"
 	"go.etcd.io/etcd/server/v3/auth"
 	"go.etcd.io/etcd/server/v3/config"
 	"go.etcd.io/raft/v3"
-	"go.uber.org/zap"
```

Comment:
I am curious why such kind of import disorder isn't discovered by https://github.com/etcd-io/etcd/blob/a4c1b3a9e2b32d64a3685ab35a39cbb62b3e08c7/.golangci.yaml#L33 ?

@demoManito

Reply:
I need to study it.

Reply:
Thank you @demoManito . Please feel free to deliver a PR to fix it if you have any findings.

Reply:
OK